frontend/public/actions/ui.ts

```diff
@@ -204,11 +203,6 @@ export const setActiveNamespace = (namespace: string = '') => {
   // otherwise users will get page refresh and cry about
   // broken direct links and bookmarks
   if (namespace !== getActiveNamespace()) {
-    const oldPath = window.location.pathname;
```

Comment:
This function is not a React component, so we cannot call the useNavigate() hook and use navigate() here. I refactored the code so this logic would run after setActiveNamespace() in other components so it can use navigate().